<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="fileNamesBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>26</height>
            </size>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="copyToClipboard_pushButton">
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="setupFilters_pushButton">
           <property name="text">
            <string>Setup Filters</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QScrollBar" name="horizontalScrollBar">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1003</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="actionCloseAll"/>
    <addaction name="separator"/>
    <addaction name="actionClear_Recent_History"/>
    <addaction name="separator"/>
    <addaction name="actionReveal_file_location"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFilesList"/>
    <addaction name="actionGraphsLayout"/>
    <addaction name="separator"/>
    <addaction name="actionFiltersLayout"/>
    <addaction name="separator"/>
    <addaction name="actionWindowOut"/>
    <addaction name="separator"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOne"/>
    <addaction name="actionZoomOut"/>
    <addaction name="separator"/>
    <addaction name="actionPlay_Pause"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrev"/>
    <addaction name="actionGo_to_start"/>
    <addaction name="actionGo_to_end"/>
    <addaction name="actionGrab_frame"/>
    <addaction name="actionGrab_plots_image"/>
    <addaction name="actionShow_hide_debug_panel"/>
    <addaction name="actionShow_hide_filters_panel"/>
    <addaction name="separator"/>
    <addaction name="actionGoTo"/>
    <addaction name="actionNavigatePreviousComment"/>
    <addaction name="actionNavigateNextComment"/>
    <addaction name="separator"/>
    <addaction name="actionToolbar"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGettingStarted"/>
    <addaction name="actionHowToUseThisTool"/>
    <addaction name="actionFilterDescriptions"/>
    <addaction name="actionPlaybackFilters"/>
    <addaction name="actionDataFormat"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuImport">
    <property name="title">
     <string>Import</string>
    </property>
    <addaction name="actionImport_XmlGz_Prompt"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="actionExport_XmlGz_Prompt"/>
    <addaction name="actionExport_Mkv_Prompt"/>
    <addaction name="actionExport_XmlGz_Sidecar"/>
    <addaction name="actionExport_XmlGz_SidecarAll"/>
    <addaction name="actionExport_XmlGz_Custom"/>
    <addaction name="separator"/>
    <addaction name="actionSignalServer_status"/>
    <addaction name="actionUploadToSignalServer"/>
    <addaction name="actionUploadToSignalServerAll"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuImport"/>
   <addaction name="menuOptions"/>
   <addaction name="menuExport"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionExport_XmlGz_Prompt"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOne"/>
   <addaction name="actionZoomOut"/>
   <addaction name="separator"/>
   <addaction name="actionWindowOut"/>
   <addaction name="actionUploadToSignalServer"/>
   <addaction name="actionSignalServer_status"/>
   <addaction name="actionFilesList"/>
   <addaction name="actionGraphsLayout"/>
   <addaction name="actionFiltersLayout"/>
   <addaction name="separator"/>
   <addaction name="actionGettingStarted"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionGettingStarted">
   <property name="text">
    <string>Getting started</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionHowToUseThisTool">
   <property name="text">
    <string>How to use this tool</string>
   </property>
  </action>
  <action name="actionFilterDescriptions">
   <property name="text">
    <string>Filter Descriptions</string>
   </property>
  </action>
  <action name="actionPlaybackFilters">
   <property name="text">
    <string>Playback Filters</string>
   </property>
  </action>
  <action name="actionDataFormat">
   <property name="text">
    <string>Data Format</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionGoTo">
   <property name="text">
    <string>Go to frame at position...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toolbar</string>
   </property>
  </action>
  <action name="actionGraphsLayout">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Graphs layout</string>
   </property>
  </action>
  <action name="actionFiltersLayout">
   <property name="text">
    <string>Filters layout</string>
   </property>
  </action>
  <action name="actionFilesList">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Files list</string>
   </property>
  </action>
  <action name="actionWindowOut">
   <property name="text">
    <string>Window out</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="text">
    <string>Close all</string>
   </property>
  </action>
  <action name="actionExport_XmlGz_Prompt">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>To .qctools.xml.gz...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionImport_XmlGz_Prompt">
   <property name="text">
    <string>From .qctools.xml.gz...</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionExport_XmlGz_Sidecar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>To sidecar .qctools.xml.gz</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExport_XmlGz_Custom">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>To prefered directory</string>
   </property>
  </action>
  <action name="actionExport_XmlGz_SidecarAll">
   <property name="text">
    <string>To sidecar .qctools.xml.gz (All files)</string>
   </property>
  </action>
  <action name="actionZoomOne">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom 1:1</string>
   </property>
   <property name="toolTip">
    <string>Zoom 1:1</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionSignalServer_status">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icon/signalserver_not_uploaded.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Signalserver status: not uploaded</string>
   </property>
   <property name="toolTip">
    <string>Signalserver status: not uploaded</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionUploadToSignalServer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icon/signalserver_upload.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Upload to Signalserver</string>
   </property>
   <property name="toolTip">
    <string>Upload current stats to signalserver</string>
   </property>
  </action>
  <action name="actionUploadToSignalServerAll">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icon/signalserver_upload.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Upload to Signalserver (All files)</string>
   </property>
   <property name="toolTip">
    <string>Upload all stats to signalserver</string>
   </property>
  </action>
  <action name="actionNavigateNextComment">
   <property name="text">
    <string>Navigate to next comment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+.</string>
   </property>
  </action>
  <action name="actionNavigatePreviousComment">
   <property name="text">
    <string>Navigate to previous comment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="actionClear_Recent_History">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clear Recent History</string>
   </property>
  </action>
  <action name="actionReveal_file_location">
   <property name="text">
    <string>Reveal file location</string>
   </property>
  </action>
  <action name="actionExport_Mkv_Prompt">
   <property name="text">
    <string>To .qctools.mkv..</string>
   </property>
  </action>
  <action name="actionPlay_Pause">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Play/Pause</string>
   </property>
   <property name="shortcut">
    <string>K, Space</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="priority">
    <enum>QAction::HighPriority</enum>
   </property>
  </action>
  <action name="actionNext">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Next frame</string>
   </property>
   <property name="shortcut">
    <string>L, Right</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="priority">
    <enum>QAction::HighPriority</enum>
   </property>
  </action>
  <action name="actionPrev">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Previous frame</string>
   </property>
   <property name="shortcut">
    <string>J, Left</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="priority">
    <enum>QAction::HighPriority</enum>
   </property>
  </action>
  <action name="actionGo_to_start">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Go to start</string>
   </property>
   <property name="shortcut">
    <string>/, Ctrl+Left</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="priority">
    <enum>QAction::HighPriority</enum>
   </property>
  </action>
  <action name="actionGo_to_end">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Go to end</string>
   </property>
   <property name="shortcut">
    <string>], Ctrl+Right</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="priority">
    <enum>QAction::HighPriority</enum>
   </property>
  </action>
  <action name="actionGrab_frame">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Grab frame</string>
   </property>
  </action>
  <action name="actionGrab_plots_image">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Grab plots image</string>
   </property>
  </action>
  <action name="actionShow_hide_debug_panel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show / hide debug panel</string>
   </property>
  </action>
  <action name="actionShow_hide_filters_panel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show / hide filters panel</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
